src/AnnouncementBubble/QuotedAnnouncementBubble.tsx

```diff
@@ -38,7 +38,7 @@ export const QuotedAnnouncementBubble: React.FC<Props> = ({
   // don't attempt to read the entire hidden message. This only works for plain-string children
   let content = children;
   if (!isExpanded && typeof content === "string") {
-    content = content.substring(0, 100);
+    content = content.substring(0, 300);
```

Comment:
QQ about this: I don't think that we can actually fit 300 characters in the single line, non-expanded version of this for the quoted bubble (at least with the way it's currently implemented on the messaging page), so a screen reader would still pick up more characters than are actually visible to the user in that case.

If we need to support this increased character count for a different use-case, can we make this dynamic?

Reply:
yeah i think you are right. i was not sure a dynamic way to do this. Chloe had raised this as an issue recently and I think we left figuring out a better way to handle this as a todo. i think i had initially done a quick calculation that 100 characters should be the max based on the 14px size and the max width of the bubble, but I saw an issue in Dewey when the Bubble wasn't limited by the chat width. Although, that shouldnt be a realistic scenario, I thought that for now to be safe I would increase the limit until we do some proper calculations to figure out what these limits should be. very open to feedback!